jack.py

```diff
@@ -674,6 +675,38 @@ def transport_stop(self):
         """Stop JACK transport."""
         _lib.jack_transport_stop(self._ptr)
 
+    def set_timebase_callback(self,
+                              conditional, timebase_callback, userdata=None):
+        """Register as timebase master for the JACK subsystem.
+
+        The timebase master registers a callback that updates extended position
+        information such as beats or timecode whenever necessary. Without this
+        extended information, there is no need for this function.
+
+        There is never more than one master at a time. When a new client takes
+        over, the former timebase_callback is no longer called. Taking over the
+        timebase may be done conditionally, so it fails if there was a master
+        already.
+
+        The method may be called whether the client has been activated or not.
+
+        Parameters
+        ----------
+        conditional : int
+            non-zero for a conditional request.
+        timebase_callback : callable
+            realtime function that returns position information.
+        arg :
+            an argument for the timebase_callback function.
+        """
+        @self._callback("JackTimebaseCallback")
+        def callback_wrapper(state, nframes, pos, new_pos, _):
+            return timebase_callback(state, nframes, pos, new_pos, userdata)
+
+        _check(_lib.jack_set_timebase_callback(
+            self._ptr, conditional, callback_wrapper, _ffi.NULL),
+            "Error setting timebase callback")
```

Comment:
OK, if you need `conditional=True`, we'll have to work around the JACK2 bug.

But we still should check for `EBUSY`, because JACK1 might use it correctly and at some point JACK2 might get fixed.
Our code should just continue to work in this case.